Shared/Network/NetworkManager.swift

```diff
@@ -156,6 +156,32 @@ struct NetworkManager {
                     reservationType = .none
                 }
 
+                var expandedMenuData: [String: [ExpandedMenu.Item]] = [:]
+                var orderedCategories: [String] = []
+
+                eatery.expandedMenu.compactMap { $0 }.forEach { expandedMenu in
+                    expandedMenuData[expandedMenu.category] = []
+                    orderedCategories.append(expandedMenu.category)
+                    var expandedMenuItems: [ExpandedMenu.Item] = []
+                    let stations = expandedMenu.stations.compactMap { $0 }
+
+                    stations.forEach { station in
+                        let items = station.items.compactMap { $0 }
+
+                        items.forEach { item in
+                            let name = item.item
```

Comment:
`item.item` is sus, would be good to actually change it to `.name` in a future PR

Reply:
I believe this is actually like that because of the graphQL query needed to get the data from the backend.  So, I think this is a backend issue.

To avoid item.item, though I could name it something like 

`let itemContainers = station.items.compactMap { $0 }`

`itemContainers.forEach { itemContainer in`
       `    let name = itemContainer.item`